zepid/causal/ipw/IPMW.py

```diff
@@ -32,15 +33,21 @@ def __init__(self, df, missing_variable, stabilized=True):
         Extracting calculated weights
         >>>ipm.Weight
         """
-        if df.loc[df[missing_variable].isnull()][missing_variable].shape[0] == 0:
-            raise ValueError('IPMW requires that missing data is coded as np.nan')
+        # Checking input data has missing labeled as np.nan
+        if type(missing_variable) is str:
+            if df.loc[df[missing_variable].isnull()][missing_variable].shape[0] == 0:
```

Comment:
one slightly more convenient notation in pandas is `df.loc[ix, column]`, so this could be:
```python
df.loc[df[missing_variable].isnull(), missing_variable].shape[0]
``` 
You get to remove two brackets!

Reply:
One nice thing, (I might be wrong), is that the above syntax allows for dataframe updates:
```python 
df.loc[df[missing_variable].isnull(), missing_variable] = value  # a-okay

df.loc[df[missing_variable].isnull()][missing_variable] = value  # throws a warning and doesn't update
```